prime-dotnet-webapi-tests/Unit Tests/EmailService.Tests.cs

```diff
@@ -119,5 +105,17 @@ public static IEnumerable<object[]> RenewalScheduleTestCases()
                 }
             }
         }
+
+        [Fact]
+        public async void TestSendRemoteUserNotificationsAsync_NoRemoteUsersDoesNotThrow()
+        {
+            var service = CreateService();
+            var site = new SiteFactory().Generate();
+            var remoteUsers = Enumerable.Empty<RemoteUser>();
+
+            var exceptions = await Record.ExceptionAsync(() => service.SendRemoteUserNotificationsAsync(site, remoteUsers));
```

Comment:
@neophyte57 This is the more pragmatic way of doing a test that asserts that no exceptions are thrown. Rather than call the code and let the test exploding mean it failed, we capture thrown exceptions and assert that none were caught.